spec/requests/todos_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "API V1 ToDo", type: :request do
```

Comment:
Convention within rails would be that the test file maps to the same folder structure as the production file it is testing. At the same time the filename is usually the same and includes `_spec` appended.

So if we are testing: `/app/controllers/api/v1/todos_controller.rb` I would expect that test file to be located at: `spec/controllers/api/v1/todos_controller_spec.rb`. Most tooling will make this assumption and have helper shortcuts to "goto test file" if you have structured it correctly.